### PR TITLE
Add composer.json

### PR DESCRIPTION
**Pull request by yannickoo**
In order to support Composer we need to add a `composer.json` file :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "drupal/redirect",
+  "description": "Sandbox for Drupal 8 Redirect Port",
```
Comment (Berdir) [inline composer.json]: This will eventually move back to d.o, just give it the right name now? The module name is Redirect here as well.

Comment (yannickoo) [inline composer.json]: I thought it's common to just use `drupal/MODULE` like [Media Entity](https://github.com/drupal-media/media_entity/blob/8.x-1.x/composer.json) did it. Am I wrong here?

Comment (Berdir) [inline composer.json]: The name is fine, I'm talking about the description :)

**Comment by yannickoo** [comment]
@Berdir I have updated the description :)

**Comment by Berdir** [comment]
Merged, thanks.

**Comment by ModulesUnraveled** [comment]
I'm trying to figure out how to include this in a project. Here's what I've tried, can you show me where I'm going wrong?

```
"repositories": [
    {
        "type": "git",
        "url": "https://github.com/md-systems/redirect.git"
    }
],

"require": {
    "drush/redirect": "8.0.1@dev"
},
```

(I've also tried)

```
"require": {
    "drush/redirect": "dev-master"
},
```

**Comment by ModulesUnraveled** [comment]
Typo: Just to clarify, I tried "drupal/redirect" in the require section, not "drush/redirect"

**Comment by ModulesUnraveled** [comment]
Got it figured out! Posting here in case anyone else wants to do it:

```
"repositories": [
    {
        "type": "git",
        "url": "https://github.com/md-systems/redirect"
    }
],

"require": {
    "drupal/redirect": "dev-8.x-1.x"
},
```

**Pull request merged**